modules/app_component/main.tf

```diff
@@ -9,13 +9,16 @@
 # ---------------------------------------------------------------------------------------------------------------------
 # Service definition, auto heals if task shuts down
 # ---------------------------------------------------------------------------------------------------------------------
+locals {
+  ecs_launch_type = var.cluster_type == "ECS_FARGATE" ? "FARGATE" : "EC2"
+}
 resource "aws_ecs_service" "ecs_service" {
-  name             = "${var.name}Service"
-  cluster          = data.aws_ecs_cluster.selected.arn
-  task_definition  = aws_ecs_task_definition.ecs_task_definition.arn
-  desired_count    = var.instances
-  launch_type      = "FARGATE"
-  platform_version = "LATEST"
+  name            = "${var.name}Service"
+  cluster         = data.aws_ecs_cluster.selected.arn
+  task_definition = aws_ecs_task_definition.ecs_task_definition.arn
+  desired_count   = var.instances
+  launch_type     = local.ecs_launch_type #"FARGATE"
+  #platform_version = local.fargate_platform_version
```

Comment:
Does it need to be commented out? Otherwise, please comment it in again.

Reply:
platform_version is only available for FARGATE and not for EC2. For FARGATE, the default value of platform_version is "latest". 
So, can I just remove this line?